docs-source/spring/content/development/project.md

```diff
@@ -63,11 +71,9 @@ can take advantage of the built-in platform services:
 </dependencyManagement>    
 ```
 
-### Spring Application Configuration
+## Spring Application Configuration
 
-Oracle recommends the following configuration in order for the application access to
-use the built-in services, including the Spring Boot Eureka Service Registry and the
-observability tools:
+Oracle recommends the following configuration in order for the application access to use the built-in services, including the Spring Boot Eureka Service Registry and the observability tools:
```

Comment:
If it doesn't change your intended meaning, I suggest "Oracle recommends ... for the application to access the built-in ..."